### ISSUE TITLE
Can't paste login details when authorizing

### ISSUE THREAD

**Issue by riotherio** [jonathontoon/tweet-tray#52]
![screen shot 2018-02-04 at 3 29 19 pm](https://user-images.githubusercontent.com/3750705/35774893-2f5cf052-09c0-11e8-9a71-c62a28842dcc.png)

Just realized when i was trying to paste my password from 1password.
It looks like it doesn't respond to any paste commands or a option-click in the window.

**Comment by jonathontoon** [comment]
@riotherio did you download the latest version? I released a fix that attempted to fix that.

**Comment by riotherio** [comment]
i'm on 1.0.1, still ain't working 😢

**Comment by jonathontoon** [comment]
@riotherio cool, okay then. Which OS version are you using?

**Comment by riotherio** [comment]
I'm using high land
<img width="698" alt="screen shot 2018-02-04 at 3 52 37 pm" src="https://user-images.githubusercontent.com/3750705/35775015-720102ce-09c3-11e8-9f71-dd88bd2b56a9.png">

**Comment by riotherio** [comment]
Actually, i think all keyboard commands ain't working too, like command-A (select all) and copy (command-c) when composing in the tray

**Comment by jonathontoon** [comment]
@riotherio yeah seems to be an macOS specific bug still, Windows works. macOS works fine when running dev, but not once it's packaged...

**Comment by jonathontoon** [comment]
I think this fixes #53 it. We shall wait and see.

**Comment by jonathontoon** [comment]
@riotherio check the latest release, it _should_ be all good now on macOS.
